deserialize/__init__.py

```diff
@@ -87,6 +91,26 @@ def _deserialize_list(class_reference, list_data, debug_name):
     return output
 
 
+
+def _deserialize_tuple(class_reference, tuple_data, debug_name):
+
+    if not isinstance(tuple_data, tuple):
+        raise DeserializeException(f"Cannot deserialize '{type(tuple_data)}' as a list.")
```

Comment:
Can you add a reference to `debug_name` in here? Having that set makes it much easier to debug when you are actually using the library.